adaptors/library/jobs/kobotoolbox-get-suspected-patients.js

```diff
@@ -0,0 +1,13 @@
+// Get all submissions of a specific form/asset and get the percentage of patients suspected of having covid-19
+http.get('/assets/aDReHdA7UuNBYsiCXQBr43/data/');
```

Comment:
This is much better

Why `http.get` and not `getSubmissions`?

I'd quite like an example using the `http` namespace, but not for something that can be done easier without it

Reply:
I have updated